File: Bab 6/Exercise/3. Integer.java
import java.util.Scanner;

public class Main{

    public static void main(String[] args){

        int[] input = new int[5];

        System.out.println("Masukan lima buah angka satu persatu, tekan enter setiap selesai memasukkan");

        Scanner sc = new Scanner(System.in);

        try{
            
            input[0] = sc.nextInt();
            input[1] = sc.nextInt();
            input[2] = sc.nextInt();
            input[3] = sc.nextInt();
            input[4] = sc.nextInt();
        
        }catch(Exception e){

        }

        System.out.println("Angka ke 1 = " + input[0]);
        System.out.println("Angka ke 2 = " + input[1]);
        System.out.println("Angka ke 3 = " + input[2]);
        System.out.println("Angka ke 4 = " + input[3]);
        System.out.println("Angka ke 5 = " + input[4]);
    }
}

File: Bab 6/Exercise/3. Integer.c
#include <stdio.h>

int main(){

    int input[5];

    printf("Masukan lima buah angka secara berurutan, lalu tekan enter setelahnya\n");

    scanf("%d", &input[0]);
    scanf("%d", &input[0]);
    scanf("%d", &input[0]);
    scanf("%d", &input[0]);
    scanf("%d", &input[0]);

    printf("Angka ke - 1 = %d\n", input[0]);
    printf("Angka ke - 2 = %d\n", input[1]);
    printf("Angka ke - 3 = %d\n", input[2]);
    printf("Angka ke - 4 = %d\n", input[3]);
    printf("Angka ke - 5 = %d\n", input[4]);

    return 0;
}